keyboards/dz60/keymaps/coppertop/keymap.c

```diff
@@ -0,0 +1,80 @@
+#include QMK_KEYBOARD_H
```

Comment:
Could you add a license header here, please?

Reply:
I can but I need an example; the base map I based my keymap off of had the same first line I had.

Reply:
https://github.com/qmk/qmk_firmware/blob/410d09675a58f1890c24068066bb8eace21c2416/quantum/template/base/keymaps/default/keymap.c#L1-L15

Reply:
OK; added to both keymaps.

Reply:
Done